src/common/database.py

```diff
@@ -38,9 +42,8 @@ def get_connection(args):
         if "rds_database_name" in args
         else os.environ["RDS_DATABASE_NAME"],
         ssl_ca=abs_file_path,
-        ssl_verify_cert=not args["skip_ssl"]
-        if "skip_ssl" in args
-        else ("SKIP_SSL" not in os.environ),
+        ssl_verify_cert=False if skip_ssl else True,
```

Comment:
why have a separate var that does the same thing as `skip_ssl` and always has the same value?

Reply:
Reverted this back to the initial code